core/include/detray/navigation/intersection/helix_line_intersector.hpp

```diff
@@ -89,6 +87,9 @@ struct helix_intersector_impl<line2D<algebra_t>, algebra_t> {
         // @NOTE We might not have to call this which is meant to be for ray
         // intersection...
         if (denom < 1e-5f) {
+            std::cout
```

Comment:
This should be in an `#ifdef NDEBUG`, as it is a device function and this is not supported on all of our device platforms.

Reply:
Good point